apps/mobile/src/app/components/Auction/AuctionPage.tsx

```diff
@@ -3,21 +3,32 @@ import AuctionTemplate from './AuctionTemplate';
 import { useParams } from 'react-router-native';
 import { useQuery } from '@tanstack/react-query';
 import { apiRoutes, routeWithId } from '@inheartive/data';
+import { ScrollView } from '@inheartive/ui/atoms';
+import { Text } from '@inheartive/ui/atoms';
 
 export function AuctionPage() {
   const params = useParams();
   const { id } = params;
-
   const {
     isLoading,
     error,
-    data: auction,
+    data: auctions = [],
   } = useQuery({
     queryKey: ['auction', id],
-    queryFn: () => fetch(routeWithId(apiRoutes.auction, id)).then((res) => res.json()),
+    queryFn: () => (id ? fetch(routeWithId(apiRoutes.auction, id)).then((res) => res.json()) : []),
   });
 
-  return <AuctionTemplate isLoading={isLoading} isError={!!error} auction={auction} />;
+  if (auctions.length === 0) {
+    return (
+      <ScrollView>
+        <Text>Something went wrong</Text>
```

Comment:
auctions.length === 0 does not mean that something went wrong, right? It's just empty auctions list

Reply:
If user clicks on existing auction and goes to auction details page, this auction should exist.

Reply:
yes it should be different endpoint for one auction, just /auctions/:id

Reply:
You deleted this at al but I think It would be good to display something when list will be empty for example "Nothing found" or something like that

Reply:
I have another observation. I didn't analize that deeply but I think something is wrong here. We have two components regarding auctions. **Auction** and **Auctions**. This change above was in Auction component that should be responsible for single auction not for the list of piece of it. Maybe i don't see everything clear. In that case please explain what is wrong in my annalyze